dispatcher/backend/src/routes/utils.py

```diff
@@ -135,3 +137,59 @@ def remove_url_secrets(response: dict):
                     )
                 )
                 response[key] = response[key].replace(url, secured_url)
+
+
+def get_sort_field_and_apply_order(
+    model, sort_by, sort_order, stmt, join_models=None, fallback_field=None
+):
+    """
+    Determines the sort field based on the model and sort_by parameter,
+    then applies the sort order to the statement.
+    """
+    join_models = join_models or {}
+
+    def apply_default_sort():
```

Comment:
This is way better managed outside this function so that we can keep custom order criteria as before, no need for that in this function